tests/integration/idmap.bats

```diff
@@ -0,0 +1,227 @@
+#!/usr/bin/env bats
+
+load helpers
+
+function setup() {
+	# idmap mounts requires root and kernel >= 5.12
```

Comment:
Comments should explain not what we are doing, but why we're doing that. In here, it is obvious from the below two lines that we require this and that. So I guess this comment can be safely removed.